### ISSUE TITLE
What's the batch size and epoch considered for training ?

### ISSUE THREAD

**Issue by jaskirat111** [haydengunraj/COVIDNet-CT#3]

**Comment by haydengunraj** [comment]
For batch size, I would recommend at least 8. The epochs will depend on what batch size and learning rate you use, and whether you are fine-tuning or attempting to train from scratch. The training script performs intermittent validation, so you can watch for overfitting by tracking the validation metrics.

**Comment by jaskirat111** [comment]
I am actually using Nvidia Tesla V100 4x GPU for training from scratch with a Batch size of 32 , learning rate of 0.001 and an epoch of 200 . Please let me know your suggestions for adjusting the parameters for obtaining decent  results while training from scratch.

**Comment by haydengunraj** [comment]
It's difficult to recommend a particular strategy for training from scratch due to the way these models were developed. Following [COVIDNet](https://arxiv.org/abs/2003.09871), we employ a human-machine collaborative design strategy, and as such the final architectures are the result of the machine-driven design stage. The final architectures were derived from human-designed baselines, and only the baseline models were trained from scratch.

With that said, the settings you have chosen seem reasonable. You may end up overfitting after 200 epochs, so keep an eye on the validation metrics. If you are able to achieve reasonable test results, please add a comment here with the training settings you used so that others can use them as well.
